### ISSUE TITLE
env vars and connection params

### ISSUE THREAD

**Issue by ikennamax** [classEze/max-assessment#5]
The readme says that we should create an env file and set a connection url. Are we supposed to setup our own db and connection url? You should let us have the connection url to use, but it doesn't seem that you did that. So even if the project could run, I reckon we'd still have a db connection error. Correct?

**Comment by classEze** [comment]
Yes. That is true. Sharing connection strings to my database would defeat the purpose of using a dotenv file in the first place. I imagined, to run the program locally you would have to create your own local database and perharps my table schema and the connection string to your database would be all you need to get up and running.
